WooCommerce/Classes/ViewRelated/Dashboard/Analytics Hub/AnalyticsTimeRangeCard.swift

```diff
@@ -0,0 +1,61 @@
+import SwiftUI
+
+struct AnalyticsTimeRangeCard: View {
+
+    let timeRangeTitle: String
+    let currentRangeDescription: String
+    let previousRangeDescription: String
+
+    var body: some View {
+        VStack(alignment: .leading, spacing: Layout.verticalSpacing) {
+            HStack {
+                ZStack(alignment: .center) {
+                    Circle()
+                        .fill(Color(UIColor.systemGray6))
+                        .frame(width: Layout.calendarCircleWidth)
+                    Image(uiImage: .calendar)
+                }
```

Comment:
This works great but what do you think about this alternative?
```swift
Image(uiImage: .calendar)
    .padding()
    .background(
        Circle().foregroundColor(Color(.systemGray6))
    )
```

I think it has some benefits like
- The hierarchy is more clear. We easily now that we are adding a background for the view.
- We close the possibility for someone to add stuff in the main `HStack` by mistake.

What do you think?

Reply:
That's an awesome suggestion. Thanks for letting me know about this option! 🙇  It's fixed here: https://github.com/woocommerce/woocommerce-ios/pull/8159/commits/066840a7f1c64099fc25d2d4980a1f9a613785ef